src/pages/settings/Subscription/CardSection/utils.ts

```diff
@@ -1,9 +1,140 @@
+import {fromUnixTime} from 'date-fns';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import DateUtils from '@libs/DateUtils';
+import type {Phrase, PhraseParameters} from '@libs/Localize';
 import * as SubscriptionUtils from '@libs/SubscriptionUtils';
+import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+import type {Fund} from '@src/types/onyx';
+import type IconAsset from '@src/types/utils/IconAsset';
+
+type BillingStatusResult = {
+    title?: string;
+    subtitle?: string;
+    isError?: boolean;
+    isTrialActive?: boolean;
+    isRetryAvailable?: boolean;
+    isAddButtonDark?: boolean;
+    isAuthenticatingRequired?: boolean;
+    icon?: IconAsset;
+    rightIcon?: IconAsset;
+};
+
+function getBillingStatus(
```

Comment:
This is easy to break with future changes, let's add unit test for this method?

Reply:
Yes I think a unit test could be useful here. @JKobrynski would you mind adding a unit test to add test coverage here?

Reply:
Not a problem at all, adding it now